pkgs/applications/audio/jamesdsp/default.nix

```diff
@@ -29,13 +40,28 @@ mkDerivation rec{
     })
   ];
 
-  nativeBuildInputs = [ qmake pkg-config ];
+  nativeBuildInputs = [
+    qmake
+    pkg-config
+    wrapQtAppsHook
+    copyDesktopItems
+  ] ++ lib.optional usePulseaudio [ wrapGAppsHook ];
```

Comment:
You are sure about both hooks?

Reply:
One if for QT and one if for gnome. Which one are we using here?

Reply:
> One if for QT and one if for gnome. Which one are we using here?

We should use `wrapQtAppsHook`, this is a QT application，only Pulseaudio version need gst-plugins which  need ‘wrapGAppsHook’ 

Use ‘wrapGAppsHook’ will override `wrapQtAppsHook`‘s work, It may be because I install `qt5Full` in my system, this software can actually run when `wrapQtAppsHook` become invalid

Reply:
use qtWrapperArgs  to wrap gst-plugins should be correct way